src/bin/pgcopydb/ld_transform.c

```diff
@@ -1169,6 +1169,64 @@ parseMessage(StreamContext *privateContext, char *message, JSON_Value *json)
 }
 
 
+/* append the new insert statement if targets same table as previous */
+static bool
+coalesceInsertStatements(LogicalMessageInsert *last,
+						 LogicalMessageInsert *stmt)
+{
```

Comment:
The code style we use returns false when we failed to do the operation, rather than when we decided that we didn't have anything to do. To distinguish the two things, usually we end up adding a `bool *` parameter to the function, but in this case here I believe we could also have the function decide to either coalesce to the last statement or append to the list of statements.